<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebayerParametersControlWidget</class> 
 <widget class="QWidget" name="DebayerParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebayerParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>Method</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="methodComboBox">
       <property name="toolTip">
         <string>Method</string>
       </property>
       <property name="currentIndex">
         <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Nearest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bilinear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AHD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fourier</string>
        </property>
       </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Bayer pos</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="bayerPosSpinBox">
       <property name="toolTip">
         <string>Bayer pos</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>numBitsOut</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="numBitsOutSpinBox">
       <property name="toolTip">
         <string>numBitsOut</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>12</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>gains</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="DoubleVectorWidget" name="gains">
       <property name="toolTip">
         <string>RGB gains</string>
       </property>
       <property name="minimum">
        <double>0.1</double>
       </property>
       <property name="maximum">
        <double>8</double>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>gamma</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="DoubleVectorWidget" name="gamma">
       <property name="toolTip">
         <string>Gamma values</string>
       </property>
       <property name="minimum">
        <double>0.2</double>
       </property>
       <property name="maximum">
        <double>2.9</double>
       </property>
     </widget>
    </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>methodComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DebayerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>bayerPosSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DebayerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>numBitsOutSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DebayerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>gains</sender>
   <signal>valueChanged()</signal>
   <receiver>DebayerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>gamma</sender>
   <signal>valueChanged()</signal>
   <receiver>DebayerParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
  <customwidget>
   <class>DoubleVectorWidget</class>
   <extends>QWidget</extends>
   <header>widgets/vectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
</ui>
